proposals/p2240.md

```diff
@@ -81,81 +91,53 @@ P1858R2 has been especially influential for this proposal.
 
 Swift has very recently adopted a
 [proposal](https://github.com/apple/swift-evolution/blob/main/proposals/0393-parameter-packs.md)
-to support _definition-checked_ generic variadics. There have been several
+to support definition-checked generic variadics. There have been several
 attempts to add such a feature to Rust, but that work is
 [currently inactive](https://github.com/rust-lang/rfcs/issues/376#issuecomment-830034029).
 
 ## Proposal
```

Comment:
I feel like this part of the proposal and variadics.md necessarily overlap a lot. Would it make sense to combine Proposal and Examples sections into variadics.md and just put a pointer to that from here, like you do for the details section? (or only have the proposal for now and worry about extracting the text into variadics.md as a separate step)? For instance, the examples here would be very helpful context to include in variadics.md.

Reply:
I agree that the redundancy is awkward, but replacing this stuff with a pointer to variadics.md seems potentially awkward too, since variadics.md might change in the future. I'm especially hesitant to remove the examples from this document, because they're cited repeatedly (and changing them in variadics.md wouldn't even require a proposal).

I've copied the examples into variadics.md.

Reply:
(as suggested in my other comment:) I think this would be solved by moving all the content into p2240.md and deleting variadics.md. That would give much more context to the content in variadics.md and avoid having to duplicate anything or have pointers.